src/liberty/devCommands.ts

```diff
@@ -490,28 +490,30 @@ export async function openReport(reportType: string, libProject?: LibertyProject
         if (path !== undefined) {
             let report: any;
             if (libProject.getContextValue() === LIBERTY_MAVEN_PROJECT || libProject.getContextValue() === LIBERTY_MAVEN_PROJECT_CONTAINER) {
-                report = Path.join(path, "target", "site", reportType + "-report.html");
+                report = getReportFile(path, "reports", reportType + ".html");
             } else if (libProject.getContextValue() === LIBERTY_GRADLE_PROJECT || libProject.getContextValue() === LIBERTY_GRADLE_PROJECT_CONTAINER) {
                 report = await getGradleTestReport(libProject.path, path);
             }
             let reportTypeLabel = reportType;
             if (reportType === "gradle") {
                 reportTypeLabel = "test";
             }
-            fs.exists(report, (exists) => {
-                if (exists) {
-                    const panel = vscode.window.createWebviewPanel(
-                        reportType, // Identifies the type of the webview. Used internally
-                        libProject.getLabel() + " " + reportTypeLabel + " report", // Title of the panel displayed to the user
-                        vscode.ViewColumn.Two, // Open the panel in the second window
-                        {}, // Webview options
-                    );
-                    panel.webview.html = getReport(report); // display HTML content
-                } else {
-                    const message = localize("test.report.does.not.exist.run.test.first", report);
-                    vscode.window.showInformationMessage(message);
+            /*
+            if its a maven project, check for the report in the initial path , else try for the alternate one and if it doesnt exist in both 
+            display a prompt for its non existance
+            */
+            if(libProject.getContextValue() === LIBERTY_MAVEN_PROJECT || libProject.getContextValue() === LIBERTY_MAVEN_PROJECT_CONTAINER){
+                console.log("report path ::"+report)
+                if(!await checkReportAndDisplay(report, reportType, reportTypeLabel, libProject, "reports")){
+                    report = getReportFile(path, "site", reportType + "-report.html");
+                    await checkReportAndDisplay(report, reportType, reportTypeLabel, libProject, "site")     
                 }
-            });
+            /*
+            if its a gradle project, then check existance for the report in the path set and if not display prompt for its non existance
+            */
+            }else 
+                await checkReportAndDisplay(report, reportType, reportTypeLabel, libProject,"")
```

Comment:
Use semicolon to be consistent with the rest of the project

```suggestion
                await checkReportAndDisplay(report, reportType, reportTypeLabel, libProject,"");
```